components/pixels/Cargo.toml

```diff
@@ -10,6 +10,7 @@ rust-version.workspace = true
 [lib]
 name = "pixels"
 path = "lib.rs"
+test = false
```

Comment:
This will make `cargo` not execute the unit tests in this crate right? Why do you need to change this file at all?